Update transparantie.md (#354)
MinBZK/Algoritmekader: docs/onderwerpen/transparantie.md

---
title: Transparant zijn over algoritmes
summary:
---

# Transparant zijn over algoritmes

Overheden moeten transparant zijn over hun algoritmes. Dit betekent dat je duidelijke informatie geeft over de algoritmes die je ontwikkelt of gebruikt.

## Wanneer ben je transparant?
Je bent transparant over algoritmes en AI als je duidelijk uitlegt:

> * dat je algoritmes ontwikkelt of gebruikt

Reviewer: ```suggestion

* dat je algoritmes ontwikkelt of gebruikt
```

* welke data je hiervoor gebruikt
* hoe uitkomsten tot stand komen
* hoe belanghebbenden bezwaar kunnen maken tegen gevolgen van het algoritme

## Belang van transparantie
Als je open bent over de algoritmes die je ontwikkelt of gebruikt, kunnen burgers en bedrijven zich beter verdedigen tegen mogelijke nadelige gevolgen. Verkeerd gebruik van algoritmes kan iemands leven namelijk ernstig beïnvloeden. Bijvoorbeeld door discriminatie of een besluit dat niet klopt.

Door uit te leggen hoe het algoritme werkt, kun je de beslissingen van het algoritme makkelijker controleren. Je leert sneller waarom het bepaalde keuzes maakt en waar de zwakke plekken zitten.

Ook je organisatie is makkelijker te controleren. Omdat je transparant bent over algoritmes, kunnen burgers feedback geven. Journalisten zien wat je doet. En andere overheden kunnen hiervan leren.

## Aanpak transparant werken
Hoe je transparantie organiseert, hangt af van:

* het doel van je algoritme
* het soort algoritme waarmee je werkt
* wie de gebruikers zijn van het algoritme, bijvoorbeeld medewerkers of burgers
* de vereisten waar je aan moet voldoen
* de maatregelen die je neemt
* de doelgroep die je wil bereiken

> * de [levenscyclus-fase](https://minbzk.github.io/Algoritmekader/levenscyclus/) van je algoritme

Reviewer: ```suggestion
* de [levenscyclus-fase](../levenscyclus/index.md) van je algoritme
```

> Onderzoek goed welk [soort algoritme](https://minbzk.github.io/Algoritmekader/overhetalgoritmekader/soorten-algoritmes/) je gebruikt of wil gebruiken. Hoe groter de impact en het risico, hoe strenger de vereisten.

Reviewer: ```suggestion
Onderzoek goed welk [soort algoritme](../overhetalgoritmekader/soorten-algoritmes.md) je gebruikt of wil gebruiken. Hoe groter de impact en het risico, hoe strenger de vereisten.
```

> De keuze voor het soort algoritme bepaalt ook hoe transparant je kunt zijn. Van rekenregels kun je namelijk precies uitleggen hoe deze tot een beslissing komen. Maar complexe AI-systemen zijn vaak een black box: niemand weet precies hoe deze systemen beslissingen maken. Volledige transparantie is dan niet mogelijk. In dat geval moet je de werking zo goed mogelijk onderzoeken. Probeer bijvoorbeeld in elk geval ernstige gevolgen zoals [discriminatie](https://minbzk.github.io/Algoritmekader/onderwerpen/bias-en-non-discriminatie/) te voorkomen. 

Reviewer: ```suggestion
De keuze voor het soort algoritme bepaalt ook hoe transparant je kunt zijn. Van rekenregels kun je namelijk precies uitleggen hoe deze tot een beslissing komen. Maar complexe AI-systemen zijn vaak een black box: niemand weet precies hoe deze systemen beslissingen maken. Volledige transparantie is dan niet mogelijk. In dat geval moet je de werking zo goed mogelijk onderzoeken. Probeer bijvoorbeeld in elk geval ernstige gevolgen zoals [discriminatie](bias-en-non-discriminatie.md) te voorkomen. 
```

| Medewerker | contactpersonen, algoritmegebruik binnen de organisatie, afspraken over algoritmemanagement (ook wel: algoritmegovernance) | intern algoritmeregister, trainingen |
| Belanghebbende (iemand voor wie het algoritme gevolgen heeft) | hoe een besluit tot stand kwam, mogelijkheden om bezwaar te maken, contactmogelijkheden | brief over het besluit, webpagina over het algoritme |
| Geïnteresseerde burger | algoritmegebruik binnen de organisatie| webpagina over het algoritme, algoritmeregister |
| Auditor | werking, ontwikkelproces, keuzes van het ontwikkelteam, hoe uitkomsten tot stand komen| algoritmeregister, technische documentatie, programmeercode |
| Onderzoeker of journalist | algoritmegebruik binnen de organisatie | algoritmeregister, technische documentatie, code |

## Vereisten

<!-- list_vereisten onderwerp/transparantie no-search no-onderwerp no-rol no-levenscyclus -->

## Aanbevolen maatregelen

<!-- list_maatregelen onderwerp/transparantie no-search no-onderwerp no-rol no-levenscyclus -->

## Hulpmiddelen

* [Impact Assessment Mensenrechten en Algoritmes](https://minbzk.github.io/Algoritmekader/instrumenten/IAMA/index.html)

> * [Algoritmeregister](https://minbzk.github.io/Algoritmekader/instrumenten/algoritmeregister/index.html)

Reviewer: ```suggestion
* [Algoritmeregister](../voldoen-aan-wetten-en-regels/hulpmiddelen/algoritmeregister.md)
```

<!-- list_hulpmiddelen onderwerp/transparantie no-search no-onderwerp no-rol no-levenscyclus -->

## Help ons deze pagina te verbeteren

> Deel je idee, suggestie of opmerking via [GitHub](https://github.com/MinBZK/Algoritmekader/edit/main/docs/onderwerpen/transparantie/index.md) of mail ons via [algoritmes@minbzk.nl](mailto:algoritmes@minbzk.nl).

Reviewer: ```suggestion
Deel je idee, suggestie of opmerking via [GitHub](https://github.com/MinBZK/Algoritmekader/issues/new/choose) of mail ons via [algoritmes@minbzk.nl](mailto:algoritmes@minbzk.nl).
```

---
title: Transparant zijn over algoritmes
summary:
---

# Transparant zijn over algoritmes

Overheden moeten transparant zijn over hun algoritmes. Dit betekent dat je duidelijke informatie geeft over de algoritmes die je ontwikkelt of gebruikt.

## Wanneer ben je transparant?

> Je bent transparant over algoritmes en AI als je duidelijk uitlegt:

Reviewer 2: Onderstaande zaken bevatten de kern (mooi!) maar geen limitatieve opsomming.

> Je bent transparant over algoritmes en AI als je duidelijk uitlegt:

Reviewer 2: ```suggestion
Je draagt bij aan transparantie over algoritmes als je duidelijk uitlegt:
```

Author: Lastig, want met 'draagt bij' geven we nog maar deels antwoord op de vraag 'Wanneer ben je transparant?' Hoe kunnen we wel het hele antwoord geven? Of zullen we deze alinea anders verder afstemmen in de volgende release?

* welke code of programmeertaal je hiervoor gebruikt
* welke data je hiervoor gebruikt
* hoe uitkomsten tot stand komen
* hoe belanghebbenden bezwaar kunnen maken tegen gevolgen van het algoritme

## Belang van transparantie
Als je open bent over de algoritmes die je ontwikkelt of gebruikt, kunnen burgers en bedrijven zich beter verdedigen tegen mogelijke nadelige gevolgen. Verkeerd gebruik van algoritmes kan iemands leven namelijk ernstig beïnvloeden. Bijvoorbeeld door discriminatie of een besluit dat niet klopt.

Door uit te leggen hoe het algoritme werkt, kun je de beslissingen van het algoritme makkelijker controleren. Je leert sneller waarom het bepaalde keuzes maakt en waar de zwakke plekken zitten.

Ook je organisatie is makkelijker te controleren. Omdat je transparant bent over algoritmes, kunnen burgers feedback geven. Journalisten zien wat je doet. En andere overheden kunnen hiervan leren.

## Aanpak transparant werken
Hoe je transparantie organiseert, hangt af van:

* het doel van je algoritme
* het soort algoritme waarmee je werkt
* wie de gebruikers zijn van het algoritme, bijvoorbeeld medewerkers of burgers
* de vereisten waar je aan moet voldoen

> * de maatregelen die je neemt

Reviewer 2: Het organiseren van transparantie is een maatregel op zichzelf.

> * de maatregelen die je neemt

Reviewer 2: ```suggestion
* wie de belanghebbenden zijn
```

Author: Helder! Prima wat mij betreft.
